<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>menu</class>
 <widget class="QMainWindow" name="menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>986</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:22pt; color:#ffffff;&quot;&gt;Welcome To The MusicPlayer&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_logIn">
        <property name="cursor">
         <cursorShape>OpenHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(100, 203, 203);</string>
        </property>
        <property name="text">
         <string>Login</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_signUp">
        <property name="cursor">
         <cursorShape>OpenHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(100, 203, 203);</string>
        </property>
        <property name="text">
         <string>SignUp</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_forPass">
        <property name="cursor">
         <cursorShape>OpenHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(100, 203, 203);</string>
        </property>
        <property name="text">
         <string>ForgetPassword</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_exit">
        <property name="cursor">
         <cursorShape>OpenHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(100, 203, 203);</string>
        </property>
        <property name="text">
         <string>Exit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
